charts/corda-dev-prereqs/templates/tests/kafka-test.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   containers:
     - name: "kafka"
-      image: "{{ .Values.imageRegistry }}/confluentinc/cp-kafka:7.2.3"
```

Comment:
Not sure why this was left as 7.2.3 instead of 7.3.3 - if this is intentional I can revert

Reply:
No - not intentional. Looks like I bumped the version in the stateful set when I added persistence but failed to change this one.